packages/circuits/test/proveDailyClaim.test.ts

```diff
@@ -0,0 +1,197 @@
+import { CircuitConfig } from '@unirep/circuits'
+import * as utils from '@unirep/utils'
+import { IncrementalMerkleTree as zkIncrementalMerkleTree } from '@zk-kit/incremental-merkle-tree'
+import { expect } from 'chai'
+import { poseidon2 } from 'poseidon-lite'
+import { DailyClaimProof } from '../src'
+import { UnirepSocialCircuit } from '../src/types'
+import { ProofGenerationError } from './error'
+import {
+    createRandomUserIdentity,
+    genDailyClaimCircuitInput,
+    genNullifier,
+    genProofAndVerify,
+    genReputationCircuitInput,
+} from './utils'
+
+const circuit = UnirepSocialCircuit.dailyClaimProof
+
+describe('Prove report identity in Unirep Social-TW', function () {
+    this.timeout(300000)
+    /**
+     * 1. should succeed with valid inputs
+     * 2. should revert with reputation > 0
+     * 3. should revert with wrong reputation proof
+     * 4. should revert with wrong daily nullifier
```

Comment:
Can we add other tests to test: when user get a reputation after claiming a daily rep, and the user claim again within 24 hour?

And I don't know how do you check if **the epoch last claimed** is < 1 day. (I didn't see the check in circuit).